loadsbroker/aws.py

```diff
@@ -535,6 +546,10 @@ def _locate_existing_instances(self, count, inst_type, region):
             If there's insufficient existing instances for this uuid,
             whether existing or new instances should be allocated to the
             collection.
+        :param plan: Name of the instances' plan
+        :param owner: Owner name of the instances
+        :param run_max_time: instances maximum expected run time
+            in seconds
```

Comment:
Reads a little odd, maybe, "Maximum expected run-time of instances in seconds."?